doc/anti-patterns/long-method.md

```diff
@@ -8,6 +8,10 @@ Lines of code with clarification comments are usually a sign for possible method
 * If local variables and parameters prevent a method extraction, apply **Replace Temp with Query**, **Introduce Parameter Object** or **Preserve Whole Object** commands.
 * Conditional statements or loops indicate the possibility of method extraction. Use **Decompose Conditional** command for conditional expressions and **Extract Method** for loops.
 
-***
+## Detection strategy
```

Comment:
Seems like should be named as [here](https://github.com/dart-code-checker/dart-code-metrics/pull/318/files#diff-10967fa2273203d906efa4dd738c3d61d8dcb3d162c6c8c99d60ed3023c16687R7).